.clang-format

```diff
@@ -1,4 +1,23 @@
-﻿# SPDX-License-Identifier: GPL-2.0
+﻿###############################################################################
+#
+# Copyright (C) 2022-2023 Maxim Integrated Products, Inc. (now owned by
+# Analog Devices, Inc.),
+# Copyright (C) 2023-2024 Analog Devices, Inc. All Rights Reserved. This software
+# is proprietary to Analog Devices, Inc. and its licensors.
```

Comment:
Don't forget about #941 
```suggestion
# Copyright (C) 2023-2024 Analog Devices, Inc.
```

Reply:
Thanks, the line updated for .clang-format file.